frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -27,7 +27,6 @@ local MODE = {
     book_time_to_read = 6,
     chapter_time_to_read = 7,
     frontlight = 8,
-    mem_usage = 9,
```

Comment:
Btw, it can also help compare stuff like http://thedndsanctuary.eu/index.php?topic=1853.msg66563#msg66563 without needing to build the emulator.